components/Header.tsx

```diff
@@ -61,14 +54,6 @@ export default function Header({ address, isConnected }: Props) {
           Log in
         </button>
       )}
-      <button
```

Comment:
Are we not going to support chain switching anymore?

Reply:
we decided to remove for mvp, to add back later
<img width="537" alt="Screen Shot 2023-05-30 at 11 12 05 PM" src="https://github.com/Glo-Foundation/glo-wallet/assets/6061507/d9e7b46e-4a73-411f-a1d9-81963069baae">